packages/eddsa-poseidon/tests/index.test.ts

```diff
@@ -375,6 +375,7 @@ describe("EdDSAPoseidon", () => {
 
         const signature = eddsa.signMessage(message)
 
+        expect(typeof eddsa.privateKey).toBe("string")
```

Comment:
Since you expressed that you expect the private key to be what you put in, maybe this should be `expect(eddsa.privateKey).to.eq(privateKey)`

Reply:
Makes sense!